src/Components/Charts/Samples/LineChart/LineChart.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+    LineChart,
+    Line,
+    XAxis,
+    YAxis,
+    CartesianGrid,
+    Tooltip,
+    Legend,
+    ResponsiveContainer,
+} from 'recharts';
+
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+
+import styles from './_styles';
+
+const useStyles = makeStyles(styles);
+
+const parseDataset = (datasetComponent) => {
+    const { label, children } = datasetComponent.props;
+    const dataA = children.map((child) => {
+        // eslint-disable-next-line no-shadow
+        const { label, value } = child.props;
+        return { label, value };
+    });
```

Comment:
To avoid no-shadow warning:
const { label: auxLabel, value } = child.props;
return { label : auxLabel, value };

Reply:
Changed and thanks!